### PR TITLE
docs: 📚revise general documentation and licensing

### PR DESCRIPTION
**Pull request by klaarseSICKAG**
# Pull Request

## 📖 Description

- revised doc
- revised licensing
- added acknoledgement to main 3rd party dependencies
- added Synergy logo to readme and storybook

### 🎫 Issues

- https://github.com/SickDesignSystem/Internal/issues/37
- #150

### CODE REVIEW

File: packages/components/package.json
Comment (schilchSICKAG) [inline packages/components/package.json:1]: Use MIT here

File: packages/docs/NOTICE
Comment (schilchSICKAG) [inline packages/docs/NOTICE:1]: Why is LICENSE removed but NOTICE added? Wouldn´t it make more sense to juse use MIT like everywhere else?

File: packages/docs/package.json
Comment (schilchSICKAG) [inline packages/docs/package.json:1]: As above, does not make sense here. Suggestion: Use the SEE LICENSE IN LICENSE statement where it differs from MIT and use MIT on all other occurences.

File: packages/react/README.md
Comment (schilchSICKAG) [inline packages/react/README.md:1]: This package does offer components. There is nothing like form-support in React and surely not two way data-binding (React is all about unidirectional dataflow).

Suggestion: Leave this out all together or reduce to:

- Autocompletion
- Event handling
- Typescript types (which would fit better to Auto-Completion imho, as ts will provide type hints and autocomplete).

File: packages/react/package.json
Comment (schilchSICKAG) [inline packages/react/package.json:1]: See comment defails above. Noone will install a package that has such a license. And the packages don´t need it in the first place.

File: packages/tokens/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # @synergy-design-system/tokens
 
 This package provides the design tokens that form the base for all components of the synergy design system.
-It provides multiple exports that can be consumed in applications.
+It provides multiple exports for colors, spacings, shadows, sizings, etc. that can be also consumed in other applications.
```
Comment (schilchSICKAG) [inline packages/tokens/README.md:4]: We do not offer applications, so the other may be misleading here.

File: packages/tokens/package.json
Comment (schilchSICKAG) [inline packages/tokens/package.json:1]: See comments on licenses below

**Review by burktiSICKAG — APPROVED** [review]

**Pull request merged**